### ISSUE TITLE
[CI Issue] Travis CI failed itself due to "ImportError: cannot import name 'FileStorage'"

### ISSUE THREAD

**Issue by wing328** [OpenAPITools/openapi-generator#5235]
#### Bug Report Checklist

- [ ] Have you provided a full/minimal spec to reproduce the issue?
- [ ] Have you validated the input using an OpenAPI validator ([example](https://apidevtools.org/swagger-parser/online/))?
- [ ] What's the version of OpenAPI Generator used?
- [ ] Have you search for related issues/PRs?
- [ ] What's the actual output vs expected output?
- [ ] [Optional] Bounty to sponsor the fix ([example](https://www.bountysource.com/issues/66123212-javascript-client-produces-a-wrong-object-for-a-string-enum-type-that-is-used-with-ref))

##### Description

Travis CI failed itself with the following errors:

```
Installing collected packages: filelock, toml, tox, pyflakes, entrypoints, pycodestyle, mccabe, flake8
Successfully installed entrypoints-0.3 filelock-3.0.12 flake8-3.7.9 mccabe-0.6.1 pycodestyle-2.5.0 pyflakes-2.1.1 toml-0.10.0 tox-3.14.3
WARNING: You are using pip version 19.3.1; however, version 20.0.2 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
py3 create: /home/travis/build/OpenAPITools/openapi-generator/samples/server/petstore/python-aiohttp/.tox/py3
py3 installdeps: -r/home/travis/build/OpenAPITools/openapi-generator/samples/server/petstore/python-aiohttp/requirements.txt, -r/home/travis/build/OpenAPITools/openapi-generator/samples/server/petstore/python-aiohttp/test-requirements.txt
py3 installed: aiohttp==3.6.2,aiohttp-jinja2==1.1.0,async-timeout==3.0.1,atomicwrites==1.3.0,attrs==19.3.0,certifi==2019.11.28,chardet==3.0.4,Click==7.0,clickclick==1.2.2,connexion==2.0.2,coverage==5.0.3,Flask==1.1.1,idna==2.8,idna-ssl==1.1.0,importlib-metadata==1.5.0,inflection==0.3.1,itsdangerous==1.1.0,Jinja2==2.11.1,jsonschema==2.6.0,MarkupSafe==1.1.1,more-itertools==8.2.0,multidict==4.7.4,openapi-spec-validator==0.2.8,packaging==20.1,pluggy==0.13.1,py==1.8.1,pyparsing==2.4.6,pytest==4.6.9,pytest-aiohttp==0.3.0,pytest-cov==2.8.1,pytest-randomly==1.2.3,PyYAML==5.3,requests==2.22.0,six==1.14.0,swagger-ui-bundle==0.0.2,typing-extensions==3.7.4.1,urllib3==1.25.8,wcwidth==0.1.8,Werkzeug==1.0.0,yarl==1.4.2,zipp==2.1.0
py3 run-test-pre: PYTHONHASHSEED='3714362342'
py3 run-test: commands[0] | pytest --cov=openapi_server
ImportError while loading conftest '/home/travis/build/OpenAPITools/openapi-generator/samples/server/petstore/python-aiohttp/tests/conftest.py'.
tests/conftest.py:5: in <module>
    import connexion
.tox/py3/lib/python3.6/site-packages/connexion/__init__.py:3: in <module>
    from .apis import AbstractAPI  # NOQA
.tox/py3/lib/python3.6/site-packages/connexion/apis/__init__.py:1: in <module>
    from .abstract import AbstractAPI  # NOQA
.tox/py3/lib/python3.6/site-packages/connexion/apis/abstract.py:9: in <module>
    from ..operations import make_operation
.tox/py3/lib/python3.6/site-packages/connexion/operations/__init__.py:1: in <module>
    from .abstract import AbstractOperation  # noqa
.tox/py3/lib/python3.6/site-packages/connexion/operations/abstract.py:11: in <module>
    from ..decorators.response import ResponseValidator
.tox/py3/lib/python3.6/site-packages/connexion/decorators/response.py:12: in <module>
    from .validation import ResponseBodyValidator
.tox/py3/lib/python3.6/site-packages/connexion/decorators/validation.py:9: in <module>
    from werkzeug import FileStorage
E   ImportError: cannot import name 'FileStorage'
ERROR: InvocationError for command /home/travis/build/OpenAPITools/openapi-generator/samples/server/petstore/python-aiohttp/.tox/py3/bin/pytest --cov=openapi_server (exited with code 4)
___________________________________ summary ____________________________________
ERROR:   py3: commands failed
Makefile:18: recipe for target 'test-all' failed
make: *** [test-all] Error 1
[ERROR] Failed to execute goal org.codehaus.mojo:exec-maven-plugin:1.2.1:exec (test) on project PythonAiohttpServer: Command execution failed. Process exited with an error: 2 (Exit value: 2) -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
```

##### openapi-generator version

Latest master

##### Suggest a fix

We'll comment out the test for the time being.

**Comment by wing328** [comment]
Commented out Python flask test as well due to the following errors:
```
E   ImportError: cannot import name 'cached_property'
_________ ERROR collecting openapi_server/test/test_user_controller.py _________
ImportError while importing test module '/home/travis/build/OpenAPITools/openapi-generator/samples/server/petstore/python-flask/openapi_server/test/test_user_controller.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
openapi_server/test/__init__.py:4: in <module>
    from flask_testing import TestCase
.tox/py3/lib/python3.6/site-packages/flask_testing/__init__.py:13: in <module>
    from .utils import TestCase, LiveServerTestCase
.tox/py3/lib/python3.6/site-packages/flask_testing/utils.py:29: in <module>
    from werkzeug import cached_property
E   ImportError: cannot import name 'cached_property'

----------- coverage: platform linux, python 3.6.9-final-0 -----------
Name                                                 Stmts   Miss  Cover
------------------------------------------------------------------------
openapi_server/__init__.py                               0      0   100%
openapi_server/__main__.py                               9      9     0%
openapi_server/controllers/__init__.py                   0      0   100%
openapi_server/controllers/pet_controller.py            25     25     0%
openapi_server/controllers/security_controller_.py       7      7     0%
openapi_server/controllers/store_controller.py          14     14     0%
openapi_server/controllers/user_controller.py           28     28     0%
openapi_server/encoder.py                               16     16     0%
openapi_server/models/__init__.py                        7      7     0%
openapi_server/models/api_response.py                   27     27     0%
openapi_server/models/base_model_.py                    31     31     0%
openapi_server/models/category.py                       22     22     0%
openapi_server/models/order.py                          45     45     0%
openapi_server/models/pet.py                            53     53     0%
openapi_server/models/tag.py                            22     22     0%
openapi_server/models/user.py                           52     52     0%
openapi_server/test/__init__.py                         11      8    27%
openapi_server/test/test_pet_controller.py              48     48     0%
openapi_server/test/test_store_controller.py            26     26     0%
openapi_server/test/test_user_controller.py             46     46     0%
openapi_server/typing_utils.py                          15     15     0%
openapi_server/util.py                                  56     56     0%
------------------------------------------------------------------------
TOTAL                                                  560    557     1%

!!!!!!!!!!!!!!!!!!! Interrupted: 3 errors during collection !!!!!!!!!!!!!!!!!!!!
=========================== 3 error in 1.31 seconds ============================
ERROR: InvocationError for command /home/travis/build/OpenAPITools/openapi-generator/samples/server/petstore/python-flask/.tox/py3/bin/pytest --cov=openapi_server (exited with code 2)
___________________________________ summary ____________________________________
ERROR:   py3: commands failed
Makefile:20: recipe for target 'test-all' failed
make: *** [test-all] Error 1
[ERROR] Failed to execute goal org.codehaus.mojo:exec-maven-plugin:1.2.1:exec (test) on project PythonFlaskConnexionTests: Command execution failed. Process exited with an error: 2 (Exit value: 2) -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
```

**Comment by zibuyu1995** [comment]
werkzeug==0.16.1 solved this problem.

**Comment by wing328** [comment]
@zibuyu1995 yup, that's one way to do it but can we update the code to work with the latest version of wekzeug - 1.0.0 (https://pypi.org/project/Werkzeug/#history)?

**Comment by richardwhiuk** [comment]
Upstream bug is https://github.com/zalando/connexion/issues/1149

**Comment by spacether** [comment]
For python versions < 3.6 we will need to hard code the installation of:
werkzeug==0.16.1
See the versions of connexion that we install here: https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/main/resources/python-flask/requirements.mustache are limited by python versions.
Per the upstream connexion bug, the fix only went in in connexion == 2.6.0 so earlier versions of connexion are impacted.

**Comment by spacether** [comment]
This has been resolved by https://github.com/OpenAPITools/openapi-generator/pull/5423
